src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+  const adder = (a = null) => {
```

Comment:
Are U sure U need to null the parameters?

Reply:
there is no sense in doing that